tests/unit-test/test_cipher.c

```diff
@@ -56,7 +56,7 @@ void test_cipher1(void) {
                                .keybits = TA_AES_KEY_BITS};
   // Don't forget to set the initialization vector
   memcpy(encrypt_ctx.iv, iv_global, AES_IV_SIZE);
-  TEST_ASSERT(aes_encrypt(&encrypt_ctx) == RET_OK);
+  TEST_ASSERT(aes_encrypt(&encrypt_ctx) == SC_OK);
```

Comment:
`TEST_ASSERT` can be replaced with `TEST_ASSERT_EQUAL_INT32(SC_OK, aes_encrypt(&encrypt_ctx))` here.
Take a look the tests under `/unit-test` or `/api`